presto-product-tests/src/main/java/io/prestosql/tests/iceberg/TestCreateTable.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.tests.iceberg;
+
+import io.airlift.log.Logger;
+import io.prestosql.tempto.AfterTestWithContext;
+import io.prestosql.tempto.BeforeTestWithContext;
+import io.prestosql.tempto.ProductTest;
+import org.testng.annotations.Test;
+
+import static io.prestosql.tempto.assertions.QueryAssert.assertThat;
+import static io.prestosql.tempto.query.QueryExecutor.query;
+import static io.prestosql.tests.TestGroups.CREATE_TABLE;
+
+public class TestCreateTable
```

Comment:
Can you add these tests to `TestIcebergSmoke`? Product tests should only be used if absolutely necessary, since they are harder to run (you can't run them in the IDE or easily debug them).

Reply:
Yes, moved the tests to TestIcebergSmoke. 
I was able to run product tests in IDE similar to how I run smoke, but I agree that smoke tests is a better place for these.